update blog post by DUNFAN

neozhaoliang opened the pull request:
Update the English blog post by @AmesingFlank and the code.

AmesingFlank commented:
if we're to update the code, we need to update the gifs as well. Could you please include the updated gifs?

neozhaoliang commented:
Yes the updated code used 12 points to improve boundary stability.<br/><br/>YES It's better to remove the Todo line. I will fix these and the gif later.<html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body>
<br/><br/><br/><div class="ntes-mailmaster-quote" style="padding-top: 1px; padding-bottom: 1px" >
    <div style=" margin-top: 2em; margin-bottom: 1em; font-size: 14px; line-height: 1.25; color: #78787a; " >---- 回复的原邮件 ----</div>
    <div style=" margin-bottom: 1em; font-size: 12px; line-height: 1.25; color: #232324; padding: 0.5em 0.25em; border-radius: 0.67em; background-color: #f0f0f0; " >
        <table width="100%" cellpadding="0" cellspacing="9" border="0">
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    发件人
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-from" style="text-decoration:none;color:#0886e8;" ***@***.***">Dunfan ***@***.***&gt;</a>
                </td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    日期
                </td>
                <td class="mail-date" valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    2022年01月10日 10:39
                </td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    收件人
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-to" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>
                </td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    抄送至
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-cc" style="text-decoration:none;color:#0886e8;" ***@***.***">Zhao ***@***.***&gt;</a>、<a class="mail-cc" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>
                </td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    主题
                </td>
                <td class="mail-subject" valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    Re: [taichi-dev/docs.taichi.graphics] update blog post by DUNFAN (PR #144)
                </td>
            </tr>
        </table>
    </div>
    <div><p></p>

***@***.***</b> commented on this pull request.</p>



<hr>



<p>In <a href="https://github.com/taichi-dev/docs.taichi.graphics/pull/144#discussion_r780450962">website/blog/2021/10/13/head-first-taichi.md</a>:</p>

<pre style='color:#555'>&gt;  ```

-No need to worry about the meaning behind the value each `x[i,j]` -- it is only chosen so that the cloth falls down at the 45 degrees angle as shown in the gif.

+Note that all the mass points have their `y` components equal to `0.6`, we also added a little perturbations between range `[-0.1, 0.1]` to the initial position of the cloth in the `x` and `z` direction. This would make our animation look more random.

</pre>

⬇️ Suggested change

<pre style="color: #555">-Note that all the mass points have their `y` components equal to `0.6`, we also added a little perturbations between range `[-0.1, 0.1]` to the initial position of the cloth in the `x` and `z` direction. This would make our animation look more random.

+Note that all the mass points have their `y` components equal to `0.6`. We also add little perturbations in the range `[-0.1, 0.1]` to the initial positions of the points in the `x` and `z` directions, which makes the animation look more randomized and natural.

</pre>





<hr>



<p>In <a href="https://github.com/taichi-dev/docs.taichi.graphics/pull/144#discussion_r780509083">website/blog/2021/10/13/head-first-taichi.md</a>:</p>

<pre style='color:#555'>&gt;  

-Next up, we will handle the internal forces of the strings. Firstly, notice from the previous illustration that each mass point is connected to at most 8 neighbors. These links are represented in our program as follows:

+Next up, we will handle the internal forces of the strings. For each mass point we connect it to its 12 nearest neighbors in the grid, these connections are collected into a 1d list:

</pre>

<p dir="auto">12?</p>



<hr>



<p>In <a href="https://github.com/taichi-dev/docs.taichi.graphics/pull/144#discussion_r780513373">website/blog/2021/10/13/code.py</a>:</p>

<pre style='color:#555'>&gt; \ No newline at end of file

+

+#TODO: include self-collision handling

</pre>

<p dir="auto">Better not leave a todo in the blog post</p>



<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br />Reply to this email directly, <a href="https://github.com/taichi-dev/docs.taichi.graphics/pull/144#pullrequestreview-846881770">view it on GitHub</a>, or <a href="https://github.com/notifications/unsubscribe-auth/AFR2HJXDLKJW2I3OSEF5VP3UVJBHNANCNFSM5LJPJ6KA">unsubscribe</a>.<br />Triage notifications on the go with GitHub Mobile for <a href="https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675">iOS</a> or <a href="https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub">Android</a>.

<br />You are receiving this because you authored the thread.<img src="https://github.com/notifications/beacon/AFR2HJQXYY2YRL6P5PVW3YDUVJBHNA5CNFSM5LJPJ6KKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOGJ5GH2Q.gif" height="1" width="1" alt="" /><span style="color: transparent; font-size: 0; display: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; height: 0; max-width: 0; max-height: 0; mso-hide: all">Message ID: <span>&lt;taichi-dev/docs</span><span>.</span><span>taichi</span><span>.</span><span>graphics/pull/144/review/846881770</span><span>@</span><span>github</span><span>.</span><span>com&gt;</span></span></p>

<script type="application/ld+json">[

{

***@***.***": "http://schema.org",

***@***.***": "EmailMessage",

"potentialAction": {

***@***.***": "ViewAction",

"target": "https://github.com/taichi-dev/docs.taichi.graphics/pull/144#pullrequestreview-846881770",

"url": "https://github.com/taichi-dev/docs.taichi.graphics/pull/144#pullrequestreview-846881770",

"name": "View Pull Request"

},

"description": "View this Pull Request on GitHub",

"publisher": {

***@***.***": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]</script></div>
</div>
</body>
</html>
